### PR TITLE
123 organizations list page

### PR DESCRIPTION
**Pull request by tanmaythole**
My Organizations List Page - UI design

Tasks:
- [x] Create a redux store
- [X] Setup routing
- [X] UI design
- [ ] API integration

Ticket:
https://trello.com/c/ZdcxPANj/123-multitenancy-ui-organizations-list

Screenshots:
Screenshot from 2022-12-28 17-17-38

### CODE REVIEW

File: src/constants/routeConstants.js [hunk 1]
```diff
@@ -8,6 +8,11 @@ export const SIGNUP_ROUTE = "/signup";
 // private route constants
 export const DASHBOARD_ROUTE = "/dashboard";
 export const USER_PROFILE_ROUTE = "/profile";
+export const MY_ORGANIZATIONS_ROUTE = "/organizations";
+export const ADD_ORGANIZATION_ROUTE = "/organization/new";
+
+// organizations routes
+export const ORG_OVERVIEW_ROUTE = "/orgs/:org";
```
Comment (ChinmaySonawane) [inline src/constants/routeConstants.js:15]: why this is different?
at line no 11 and 12 we are using organizations, organization and on this line we are using orgs

File: src/containers/OrganizationContainer/MyOrganizations/index.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+});
+
+const MyOrganizations = () => {
+  const dispatch = useDispatch();
+
+  const { isOrgsLoading, organizations } = useSelector(mapState);
+
+  // TO-DO: dispatch leave organization
+  const leaveOrg = useCallback((org) => {
+    console.log(org);
```
Comment (ChinmaySonawane) [inline src/containers/OrganizationContainer/MyOrganizations/index.jsx:20]: remove console log

Comment (tanmaythole) [reply src/containers/OrganizationContainer/MyOrganizations/index.jsx:20]: This task is on the todo list, need to update once API is ready

File: src/constants/routeConstants.js (hunk 1 above)
Comment (tanmaythole) [reply src/constants/routeConstants.js:15]: as we are following GitHub standards, this routing was created. I will again verify with Sameer sir and the managers and then perform an action on it.

**Pull request merged**